src/main/resources/youtube/branding/xisr_yellow/monochrome/drawable/adaptive_monochrome_ic_youtube_launcher.xml

```diff
@@ -1,18 +1,18 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="108dp"
-    android:height="108dp"
-    android:viewportWidth="108"
-    android:viewportHeight="108">
-  <path
-      android:pathData="M31.11,48.3C31.26,43.28 34.46,38.94 39.42,38.11C43.04,37.51 47.9,37 54,37C60.1,37 64.96,37.51 68.58,38.11C73.54,38.94 76.74,43.28 76.89,48.3C76.96,50.26 77,52.28 77,54C77,55.72 76.96,57.74 76.89,59.7C76.74,64.72 73.54,69.06 68.58,69.89C64.96,70.49 60.1,71 54,71C47.9,71 43.04,70.49 39.42,69.89C34.46,69.06 31.26,64.72 31.11,59.7C31.05,57.74 31,55.72 31,54C31,52.28 31.05,50.26 31.11,48.3Z"
-      android:strokeWidth="5"
```

Comment:
the removal of this is intended?

Reply:
the stroke is unnecessary for this icon, it can ruin the icon